hypothesis-python/tests/numpy/test_type_lookup.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis/
+#
+# Copyright the Hypothesis Authors.
+# Individual contributors are listed in AUTHORS.rst and the git log.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from hypothesis import given, strategies as st
+
+from numpy.typing import NDArray
+import numpy as np
+
+from tests.common.debug import assert_all_examples
+
+
+def test_from_numpy_ndarray_any_type():
```

Comment:
All these tests look good - they're currently failing because `hypothesis.strategies._internal.types.is_a_type()` returns False for the `NDArray` type(-like) object; fix that and we should be good to go 😁

Reply:
I added the in_np_type, not sure if this is the fix?